src/drozer/console/session.py

```diff
@@ -540,7 +540,7 @@ def help_intents(self):
 
         For Intents, use the 'intent://' format of URI:
 
-          --extra parcelable yayintentyay intent://host.com?queryparameter=value#Intent;package=targetPackage;component=targetComponent;action=action;category=category;type=type;identifier=identifier;launchFlags=launchFlags;scheme=scheme;sourceBounds=sourceBounds;S.extraString=value;B.extraBoolean=value;b.extraByte=value;c.extraChar=value;d.extraDouble=value;i.extraInt=value;f.extraFloat=value;l.extraLong=value;s.extraShort=value;end;
+          --extra parcelable yayintentyay intent://host.com?queryparameter=value#Intent;package=targetPackage;component=targetPackage/.targetComponent;action=action;category=category;type=type;identifier=identifier;launchFlags=launchFlags;scheme=scheme;sourceBounds=sourceBounds;S.extraString=value;B.extraBoolean=value;b.extraByte=value;c.extraChar=value;d.extraDouble=value;i.extraInt=value;f.extraFloat=value;l.extraLong=value;s.extraShort=value;end;
```

Comment:
Realistically, we will need better documentation **with examples** somewhere else, but for now:

- I'd suggest we drop the mention of `package` here - you can achieve the same results with just `component` (I can't think of a case where specifying both makes sense)
- I'd write it down as `targetPackage/targetComponent` - I'd argue that (when it applies) the leading dot is part of the name of the target component. `com.somepackage/com.somepackage.somecomponent` or `com.somepackage/org.something.else.entirely` would both be perfectly valid components, too